src/storage/src/hummock/local_version/local_version_manager.rs

```diff
@@ -239,20 +241,18 @@ impl LocalVersionManager {
         if wait_epoch == HummockEpoch::MAX {
             panic!("epoch should not be u64::MAX");
         }
+
+        {
+            let current_version = self.local_version.read();
+            if current_version.pinned_version().max_committed_epoch() >= wait_epoch {
+                return Ok(());
+            }
+        }
+
```

Comment:
I think there is a bug here.
If `current_version` changes before we fetch receiver, and `current_version` never changes any more after we fetch receiver, then we will be in infinite wait.

Reply:
Thanks, fix it